src/main/resources/application-local.yml

```diff
@@ -9,11 +9,13 @@ spring:
         show-sql: true
         format_sql: true
 
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    url: ${SPRING_DATASOURCE_URL}
+    username: ${SPRING_DATASOURCE_USERNAME}
+    password: ${SPRING_DATASOURCE_PASSWORD}
```

Comment:
please, move this back to `application,yaml` 
see comment for that file with the justification for it.

Reply:
Ok.